tutor/commands/images.py

```diff
@@ -54,6 +54,12 @@ def images_command() -> None:
     "--target",
     help="Set the target build stage to build.",
 )
+@click.option(
+    "--docker-arg",
```

Comment:
Could we maybe add a short form version of this argument? maybe `-d`?

Reply:
Added short form ``-d``.